content/blog/2017/07/2017-07-13-speaker-blog-rosetta-stone.adoc

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: "Automated Software Maintenance"
+tags:
+- event
+- jenkinsworld2017
+author: hinman
+---
+
+NOTE: This is a guest post by Kevin Burnett, DevOps Lead at
+link:http://rosettastone.com[Rosetta Stone].
+
+Have you experienced that thing where you make a change in an app, and when you
+go to check on the results of the build, you find an error that really doesn't
+seem relevant to your change? And then you notice that your build is the first
+in over a year. And then you realize that you have accidentally become the
+subject matter expert in this app.
+
+You have no clue what changed (or when) to cause the failure. Did a continuous
+integration build agent become a
```

Comment:
this sounds weird as written, perhaps s/continuous integration build agent/Jenkins agent/

(this is the jenkins blog after all, we can name names :smiling_imp: )